.github/workflows/discord_notify.yml

```diff
@@ -0,0 +1,17 @@
+name: Discord Notify
+
+on:
+  pull_request:
+    types: [opened]
+    
+jobs:
+  notify:
+      if: ${{ github.event.label.name == 'React Core Team' }}
```

Comment:
Isn't that going to create a race between this workflow and the workflow adding the label? There's a separate event for labels so that's probably more robust to use but will trigger more workflow runs.

Reply:
Yeah, I'm going to test this in my fork to get the event right. I think it needs to be on PR labeled.